.travis.yml

```diff
@@ -14,7 +14,7 @@ install:
   - pip install -e . --cache-dir $HOME/.pip-cache/
 script:
   - flake8 molo
-  - molo scaffold testapp --include=molo.yourtips ^yourtips/
+  - molo scaffold testapp --include=molo.yourtips ^yourtips/ --include=likes ^likes/
```

Comment:
@cblignaut is there any specific reason that we are not including the likes URL in the urls.py, instead of adding it here?